Automattic/newspack-newsletters
# Remove custom preview button

### All Submissions:






### Changes proposed in this Pull Request:

A few things to note:
* Currently we have 3 different means of previewing: Editor, Mailchimp, Test Email
* The current "Preview" button (custom modal) is different from Gutenberg's preview (editor screen being resized) and is not located in the editor's toolbar
* Editor styles are getting closer to the final rendering (except for the block margins which are needed since you're still in the editor and you can still move blocks around)
* Mailchimp preview doesn't necessarily look 100% exactly like the final email with a different font family
* What will happen when we add more providers?

All of this makes me think we should simplify the plugin, remove the custom preview and simply rely on Gutenberg's default preview which is more than enough. The real true preview -- and what users tend to do anyway -- is via a Test Email.

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Do you see the preview dropdown in the editor toolbar?
3. Switch between desktop/tablet/mobile. Does it look ok?
4. Send a test email and compare on different devices with the preview you get from the editor. Is it acceptable? 

### Other information:
